<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSearchPathsSettingsPage</class>
 <widget class="QWidget" name="CSearchPathsSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="searchPathsGroupBox">
     <property name="title">
      <string>Search paths</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0" rowspan="2">
       <widget class="QListWidget" name="pathsListWidget"/>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QToolButton" name="addToolButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../object_viewer_qt.qrc">
            <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="removeToolButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../object_viewer_qt.qrc">
            <normaloff>:/images/list-remove.png</normaloff>:/images/list-remove.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="upToolButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../object_viewer_qt.qrc">
            <normaloff>:/images/go-up.png</normaloff>:/images/go-up.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="downToolButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../object_viewer_qt.qrc">
            <normaloff>:/images/go-down.png</normaloff>:/images/go-down.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>195</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../object_viewer_qt.qrc"/>
 </resources>
 <connections/>
</ui>
